examples/golang/verify.sh

```diff
@@ -14,6 +14,12 @@ docker-compose -f docker-compose-go.yaml up --remove-orphans go_plugin_compile
 run_log "Start all of our containers"
 bring_up_example
 
+# remove compile image of go plugin
+go_plugin_image_id=$(docker images --filter=reference='golang-go_plugin_compile:*' --quiet)
+if [[ -n "$go_plugin_image_id" ]]; then
```

Comment:
could you see #25373 - i think that provides a more generic solution

Reply:
good. I'll cherry pick it.

Reply:
before doing that - maybe (revert this and) pick #25374 - so we can see if that also resolves